src/guess_density.jl

```diff
@@ -116,6 +116,28 @@ function gaussian_superposition(basis::PlaneWaveBasis{T}, gaussians) where {T}
     irfft(basis, ρ / sqrt(basis.model.unit_cell_volume))
 end
 
+"""
+    numeric_superposition(basis::PlaneWaveBasis)
+
+Build a superposition of functions on a radial grid as a guess for the density and
+magnetisation. Expects all the pseudopotentials of the atoms of the basis to contain
+atomic valence charge densities.
+"""
+function numeric_superposition(basis::PlaneWaveBasis{T}) where {T}
+    model = basis.model
+    ρ = zeros(complex(T), basis.fft_size)
+    for (iG, G) in enumerate(G_vectors(basis))
+        Gsq = sum(abs2, model.recip_lattice * G)
+        for group in model.atom_groups
+            element = model.atoms[first(group)]
+            form_factor = valence_charge_density_fourier(element, Gsq)
```

Comment:
This should be sqrt(Gsq), no? That might be where the discrepancy comes from?

Reply:
Yes! This was it, I think.

Core:
![Si_standard_core](https://user-images.githubusercontent.com/75646713/198249851-19a1f471-932d-4296-a7f6-c82c119a1639.gif)

Valence:
![Si_standard_valence](https://user-images.githubusercontent.com/75646713/198249867-8d99f1db-f92f-4a91-80fb-26d9b2a3a309.gif)

Reply:
Awesome!